### PR TITLE
[Impeller] Remove the impeller_enable_vulkan_playgrounds flag.

### PR DESCRIPTION
**Pull request by chinmaygarde**
Instead of failing, the test will be skipped if an incompatible driver is not
found. Along with the skip, a message detailing why the test was skipped along
with remedial actions will be printed to the console.

Earlier, these tests could not be run at all. Even if playgrounds were enabled.

### CODE REVIEW

**Comment by chinmaygarde** [comment]
Builds on top of https://github.com/flutter/engine/pull/48699 and will be rebased on top of that after it landed. But this is good for review.

File: impeller/playground/backend/vulkan/playground_impl_vk.h [hunk 1]
```diff
@@ -41,8 +43,6 @@ class PlaygroundImplVK final : public PlaygroundImpl {
   PlaygroundImplVK(const PlaygroundImplVK&) = delete;
 
   PlaygroundImplVK& operator=(const PlaygroundImplVK&) = delete;
-
-  static void InitGlobalVulkanInstance();
```
Comment (jason-simmons) [inline impeller/playground/backend/vulkan/playground_impl_vk.h:45]: If this is being removed, then also remove the declaration of `global_instance_` in `playground_impl_vk.h`

**Review by jason-simmons — COMMENTED** [review]
What is the reason for holding a `ContextVK` across test cases instead of a `VkInstance`?

Holding a `VkInstance` prevents unloading of SwiftShader while allowing each test case's `ContextVK` to be fully destroyed when the test completes.

**Comment by chinmaygarde** [comment]
@jason-simmons As I said in a previous comment, this builds on top of https://github.com/flutter/engine/pull/48699 and the context is there.

**Comment by chinmaygarde** [comment]
I guess so. I see the failures locally too though with validations enabled. So I guess they are real.

**Pull request closed without merging**